wiki/commands.md

```diff
@@ -0,0 +1,134 @@
+# Commands to know
+
+The LA metro galaxy comes with several CLI commands and their various options. This section identifies some of the most significant commands, how to use them, and where to execute them. 
+
+## Scraping data from Legistar
+
+Running the scrapers can be simple or fairly involved. You can run full scrapes or "windowed" scrapes; you can run scrapes at faster or slower rates; you can run scrapes for all data or just bills, events, or people (oh, my). 
+
+**Note!** The Metro scrapers on the server run at different intervals, depending on the day. [The crontask file includes the details](https://github.com/datamade/scrapers-us-municipal/blob/master/scripts/scrapers-us-municipal-crontask#L12), but roughly, the scrapers: 
+* run a full scrape every night
+* scrape recently updated events and bills, four times per hour, Saturday through Friday morning
+* scrape all bills and events twice per hour on Friday evenings  
+
+Sometimes, you may need to run a scrape on the server to assist with debugging ventures. If so, then consider the below commands. (N.B. Depending on the volume of the scrape, e.g., a scrape for all bills, you should consider turning off the crons before executing these commands on the server.)
+
+First, get situated on the server.
+
+```bash
+# shell into the server
+ssh ubuntu@ocd.datamade.us
+
+# visit the scrapers directory, and launch the correct virtual environment
+sudo su - datamade
+cd scrapers-us-municipal
+workon opencivicdata
+```
+
+Then, run the appropriate command.
+
+```bash
+# scrape all recently updated data
+pupa update lametro
+
+# scrape all recently updated data, but used cached pages
+pupa update lametro --fastmode
+
+# scrape all recently updated data, and move as quickly as possible (but do not use cache)
+pupa update lametro --rpm=0 
+```
+
+```bash
+# scrape bills from last 28 days
+# https://github.com/opencivicdata/scrapers-us-municipal/blob/master/lametro/bills.py#L97
+pupa update lametro bills
+
+# update all bills
+pupa update lametro bills window=0
+
+# scrape bills from the last 7 days, using cached pages
+pupa update lametro bills window=7 --fastmode
+```
+
+```bash
+# scrape all events
+# https://github.com/opencivicdata/scrapers-us-municipal/blob/master/lametro/events.py#L139
+pupa update lametro events
+
+# scrape events from the last 7 days
+pupa update lametro events window=7
+```
+
+```bash
+# scrape all people
+# the people scraper does not have a "window" argument
+# but instead determines which people to update by looking at those visible on the web interface
+pupa update lametro people
+```
+
+## Importing data to Councilmatic
```

Comment:
Hmmmm.....I sometimes do, but most often do not. Do you have a preferred practice you'd like to instate?

Reply:
hm. i do the same as you. but, i think it would be nice to have a reliable record of when the imports are run, whether it's via cron or manually. shall we default to logging?